src/main/java/picard/util/IntervalListTools.java

```diff
@@ -16,15 +16,92 @@
 
 import java.io.File;
 import java.text.DecimalFormat;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
 
 /**
- * Little class to aid working with interval lists.
+ * Performs various {@link IntervalList} manipulations.
+ *
+ * <h3>Summary</h3>
+ * This tool offers multiple interval list file manipulation capabilities, including: sorting,
+ * merging, subtracting, padding, and other set-theoretic operations. The default action
+ * is to merge and sort the intervals provided in the {@link #INPUT}s. Other options, e.g. interval subtraction, are
+ * controlled by the arguments.
+ * <br />
+ * Both {@link IntervalList} and VCF files are accepted as input. {@link IntervalList} should be denoted with the extension
+ * {@value htsjdk.samtools.util.IOUtil#INTERVAL_LIST_FILE_EXTENSION}, while a VCF must have one of {@value htsjdk.samtools.util.IOUtil#VCF_FILE_EXTENSION}, {@value htsjdk.samtools.util.IOUtil#COMPRESSED_VCF_FILE_EXTENSION},
```

Comment:
Value is not displayed, just the string "htsjdk.samtools.util.IOUtil#VCF_FILE_EXTENSION"

Reply:
This needs to be fixed in the doc-build system...